flexmeasures/ui/templates/sensors/index.html

```diff
@@ -59,15 +59,19 @@ <h3>Edit {{ sensor.name }}</h3>
                                 <h3>Upload {{ sensor.name }} data</h3>
                                 <small>Belongs to: {{ sensor.generic_asset.name }} (ID: {{ sensor.generic_asset_id }})</small>
                                 <form id="uploadForm" enctype="multipart/form-data" method="post" class="form-horizontal">
+                                    <div class="form-group pb-4">
```

Comment:
This field should be below the upload selection, which is the leading UX element.